### PR TITLE
Release notes new structure (after 2111.1?)

### PR DESCRIPTION
**Pull request by sptramer**
Signed-off-by: Stephen Tramer <stramer@amazon.com>

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Comment by sptramer** [comment]
@erickson-doug - Please review and have a 2nd signoff, can be merged to staging in a preview state. After merge make sure that SIG representatives know that they may submit PRs against `staging/2111` to get alterations to the release notes.

**Review by FiniteStateGit — APPROVED** [review]

**Comment by sptramer** [comment]
Going to close, as the RNs were put into a different process/channel than through this one.

**Pull request closed without merging**
